intellij/src/main/resources/META-INF/plugin.xml

```diff
@@ -83,6 +83,8 @@
     <extensions defaultExtensionNs="com.intellij">
         <codeInsight.lineMarkerProvider language="JAVA"
                                         implementationClass="motif.intellij.provider.ScopeNavigationLineMarkerProvider"/>
+        <codeInsight.lineMarkerProvider language="kotlin"
+                                        implementationClass="motif.intellij.provider.ScopeNavigationLineMarkerProvider"/>
         <codeInsight.lineMarkerProvider language="JAVA"
```

Comment:
Todo for parity?

Reply:
@tyvsmith can you please clarify? are you referring to add a todo for adding `ScopeHierarchyLineMarkerProvider` for kotlin?

Reply:
Yeah.